SPECS/distroless-packages/distroless-packages.spec

```diff
@@ -0,0 +1,51 @@
+Name:           distroless-packages
+Summary:        Metapackage with core sets of packages for distroless containers
+Version:        0.1
+Release:        2%{?dist}
+License:        MIT
+Group:          System Environment/Base
+URL:            http://aka.ms/cbl-mariner
```

Comment:
Please use HTTPS and ideally the final URL, not the shortcut, so the user knows where they're going.